app/uk/gov/hmrc/eoricommoncomponent/frontend/connector/SubscriptionServiceConnector.scala

```diff
@@ -48,6 +49,8 @@ class SubscriptionServiceConnector @Inject() (http: HttpClient, appConfig: AppCo
     )
     // $COVERAGE-ON
 
+    sampleData(PayloadCache.SubscriptionCreate, request)
```

Comment:
You'd be logging and exposing all of:

`ase class RequestDetail(
  SAFE: String,
  EORINo: Option[String],
  CDSFullName: String,
  CDSEstablishmentAddress: EstablishmentAddress,
  establishmentInTheCustomsTerritoryOfTheUnion: Option[String],
  typeOfLegalEntity: Option[String],
  contactInformation: Option[ContactInformation],
  vatIDs: Option[List[VatId]],
  consentToDisclosureOfPersonalData: Option[String],
  shortName: Option[String],
  dateOfEstablishment: Option[LocalDate],
  typeOfPerson: Option[String],
  principalEconomicActivity: Option[String],
  serviceName: Option[String]
)`

to an user who can Sign In via Gov Acc

Reply:
Thats a fair point. Ill add config to ensure the feature is not available in prod.

Reply:
done